waterdata/constants.py

```diff
@@ -3,48 +3,202 @@
 """
 
 STATION_FIELDS_D = {
-    'agency_cd': 'Agency',
-    'site_no': 'Site identification number',
-    'station_nm': 'Site name',
-    'site_tp_cd': 'Site type',
-    'lat_va': 'DMS latitude',
-    'long_va': 'DMS longitude',
-    'dec_lat_va': 'Decimal latitude',
-    'dec_long_va': 'Decimal longitude',
-    'coord_meth_cd': 'Latitude-longitude method',
-    'coord_acy_cd': 'Latitude-longitude accuracy',
-    'coord_datum_cd': 'Latitude-longitude datum',
-    'dec_coord_datum_cd': 'Decimal Latitude-longitude datum',
-    'district_cd': 'District',
-    'state_cd': 'State',
-    'county_cd': 'County',
-    'country_cd': 'Country',
-    'land_net_ds': 'Land net location description',
-    'map_nm': 'Name of location map',
-    'map_scale_fc': 'Scale of location map',
-    'alt_va': 'Altitude of Gage/land surface',
-    'alt_meth_cd': 'Method altitude determined',
-    'alt_acy_va': 'Altitude accuracy',
-    'alt_datum_cd': 'Altitude datum',
-    'huc_cd': 'Subbasin hydrologic unit',
-    'basin_cd': 'Drainage basin',
-    'topo_cd': 'Topographic setting',
-    'instruments_cd': 'Flags for instruments at site',
-    'construction_dt': 'Date of first construction',
-    'inventory_dt': 'Date site established or inventoried',
-    'drain_area_va': 'Drainage area',
-    'contrib_drain_area_va': 'Contributing drainage area',
-    'tz_cd': 'Time Zone abbreviation',
-    'local_time_fg': 'Site honors Daylight Savings Time',
-    'reliability_cd': 'Data reliability',
-    'gw_file_cd': 'Data-other GW files',
-    'nat_aqfr_cd': 'National aquifer',
-    'aqfr_cd': 'Local aquifer',
-    'aqfr_type_cd': 'Local aquifer type',
-    'well_depth_va': 'Well depth',
-    'hole_depth_va': 'Hole depth',
-    'depth_src_cd': 'Source of depth data',
-    'project_no': 'Project number',
+    'agency_cd': {
+        'name': 'Agency',
+        'description': 'The agency that is reporting the data.'
```

Comment:
I've gone back and forth about whether the descriptions should be here or a new lookup. This struck me as the more logical place to put it.

Reply:
I agree. We aren't using a service to get the descriptions so this seems appropriate to me.